### ISSUE TITLE
Connector Template Icon Size breaks autosave

### ISSUE THREAD

**Issue by eslateBP3** [camunda/camunda-modeler#3761]
### Describe the bug

When using a large icon file for a Connector Template, adding too many too a diagram breaks autosave and the error message does not make it clear why.

### Steps to reproduce

1. Create new Connector Template. Upload an icon file that is fairly large (I reproduced it with a 63KB file but it's also possible with something smaller). Publish the template.
2. Create new BPMN Diagram.
3. Add an activity or event and configure it to use your connector template.
4. Copy and paste that activity/event enough times to get to around 775KB (multiple artifacts by your icon file size). For me, encountered error with 13 of the 63KB files and 31 of a 25KB file).
5. Upon adding anymore of that same connector type, an error will display "Couldn't save your changes, are you offline? Please try again." and the last change is reverted.

### Expected behavior

Not sure -- maybe one of these:

1. Diagram handles the size without error.
2. Icon upload prevents files over a set size.
3. Error message indicates that issues are due to icon sizes or total diagram size hitting some limit.

### Environment

- OS: Mac OS 13.4.1 / Brave Browser 1.56.14
- Camunda Modeler Version: Web Modeler (8.2?)

**Comment by nikku** [comment]
We should probably prevent users from defining extensive icon sizes.

**Comment by nikku** [comment]
Could you share how (error message wise) the auto-safe functionality breaks? If possible, please share a screenshot that establishes the context.

**Comment by eslateBP3** [comment]
Yes. An error message is shown that states: "Couldn't save your changes, are you offline? Please try again." When this occurs, the last change is also reverted.
![image](https://github.com/camunda/camunda-modeler/assets/77014927/f95d95b2-6d6c-44e0-a0a6-fbf0b95da4dd)
